pkgs/development/interpreters/dart/update.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq
+
+set -euo pipefail
+
+CHANNELS=(stable beta dev)
+declare -A PLATFORMS=(
```

Comment:
Please be consistent.

Reply:
Regarding the `darwin` platform, or something else? I have addressed the former.

Reply:
Once you declare an array with and once without declare.

Reply:
`declare -A` declares an associative array in Bash, not a scalar array.